src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -488,22 +495,241 @@ private void testFileWithoutInfoLineInHeader(final String name, final Class<? ex
         spec.executeTest(name, this);
     }
 
-    @Test
-    public void testInvalidJexl() throws IOException {
-        final String testFile = getToolTestDataDir() + "ac0.vcf";
-
+    private void beforeJexlTest(){
         // NOTE: JexlEngine singleton construction in VariantContextUtils sets silent to false.
         // However VariantFiltration.initialize() sets setSilent(true) on the shared instance.
         // Just in case this test runs after a VariantFiltration in the same VM, always set silent back to false.
         htsjdk.variant.variantcontext.VariantContextUtils.engine.get().setSilent(false);
+    }
+
+    @Test // tsato: this test fails for the wrong reason; ac0.vcf does not exist
```

Comment:
Then could you either create it, or change the test to refer to a file that does exist?

Reply:
Fixed. It now uses the sv vcf.